src/core/debugger.ts

```
@@ -130,112 +174,470 @@
+        
+        switch (node.intentType) {
+          case IntentType.INPUT:
+            output = data[node.params['name'] as string];
+            break;
+            
+          case IntentType.CONSTANT:
+            output = node.params['value'];
+            break;
+            
+          case IntentType.FILTER:
+            const filterPred = node.params['predicate'] as Function;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)